jenkins/job-configs/kubernetes-jenkins/daily-shame-mail.yaml

```diff
@@ -0,0 +1,30 @@
+- job:
+    name: 'daily-shame-mail'
+    description: 'Send the daily flaky issue shame email. Test owner: test infra team.'
+    logrotate:
+        numToKeep: 20
+    triggers:
+        - timed: 'H 8 * * 1-5'
+    wrappers:
+        - credentials-binding:
+            - text:
+                credential-id: '8864846d-2090-4173-9b00-163a79c51913'  # read-only GitHub API token
+                variable: 'GITHUB_API_TOKEN'
+            - username-password-separated:
+                credential-id: '8184516b-02e3-426f-be2c-81296ff6d545'  # sendgrid credentials
+                username: 'SMTP_USER'
+                password: 'SMTP_PASS'
+    builders:
+        - shell: |
+            set +x  # Don't echo credentials into the build log
+            IMAGE=gcr.io/google_containers/shame-mailer:2016-04-28-cddcf3d
+            echo "Running ${IMAGE}"
+            docker run \
+              -e GITHUB_API_TOKEN="${GITHUB_API_TOKEN}" \
+              -e SHAME_FROM="kubernetes-sig-testing+flakereport@googlegroups.com" \
```

Comment:
You have permission to at a minimum update https://github.com/kubernetes/contrib/blob/master/mungegithub/reports/shame.go#L301